sdk/containerregistry/container-registry/test/public/containerRegistryClient.spec.ts

```diff
@@ -50,11 +50,10 @@ describe.skip("ContainerRegistryClient functional tests", function() {
   // NOTE: use of "function" and not ES6 arrow-style functions with the
   // beforeEach hook is IMPORTANT due to the use of `this` in the function
   // body.
-  beforeEach(function(this: Context) {
+  beforeEach(/** @this*/ function(this: Context) {
```

Comment:
Is the `@this` annotation needed in cases where we specify the type of `this` and use the function syntax - such as this example (and the note above)?

Reply:
Yes. I think the rule overlooks the names of the arguments.